resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -189,6 +189,15 @@ package object config extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("1s")
 
+  private[spark] val CLIENT_REPORT_INCLUDE_DRIVER_LOGS_LINK =
+    ConfigBuilder("spark.yarn.report.includeDriverLogsLink")
+      .doc("In cluster mode, whether print include links to the driver container's logs alongside "
```

Comment:
whether the client application report includes links to the driver container's logs.

Reply:
Thanks, looks like I wrote this too fast.